lib/assert.js

```diff
@@ -47,16 +40,11 @@ const assert = module.exports = ok;
 
 // All of the following functions must throw an AssertionError
 // when a corresponding condition is not met, with a message that
-// may be undefined if not provided.  All assertion methods provide
+// may be undefined if not provided. All assertion methods provide
 // both the actual and expected values to the assertion error for
 // display purposes.
 
-function fail(actual, expected, message, operator, stackStartFunction) {
-  if (arguments.length === 1)
-    message = actual;
-  if (arguments.length === 2)
-    operator = '!=';
-  const errors = lazyErrors();
+function _fail(actual, expected, message, operator, stackStartFunction) {
```

Comment:
I've been looking around `/lib/` there is no standard, but IMHO `_name` functions are _usualy_ exposed-but-considered-private. You can give this one a more explicit name like, `failImpl` or `failInner`